.changeset/neat-cars-reply.md

```diff
@@ -0,0 +1,5 @@
+---
+"@frontify/fondue-components": patch
+---
+
+feat: add option to position dialog at the top of the screen
```

Comment:
```suggestion
"@frontify/fondue-components": minor
---

feat: add option to position dialog at the top/bottom of the screen
```

Reply:
Don't forget the > or just top?
Because if top/bottom, then we need the bottom CSS too

Reply:
adjusted